src/locales/en-US/list.ftl

```diff
@@ -14,7 +35,7 @@ error-notification-duplicate =
 error-notification-duplicate-link = <a>Go to existing entry?</a>
 
 all-items-get-started =
-  When you save a password in Firefox, it will show up here.
+  When you save a password in { -brand-name }, it will show up here.
```

Comment:
If we're using spaces inside placeables, it would be nice to be consistent across the file, e.g.

```
error-notification-duplicate =
  An entry for { $title } with that username already exists.
```

GH doesn't allow me to comment on that line 💩